types/enzyme/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Enzyme 3.10
+// Type definitions for Enzyme 3.11
```

Comment:
please don't change this unless you're adding types for the things in 3.11 that aren't in 3.10.
```suggestion
// Type definitions for Enzyme 3.10
```